src/averageAges.js

```diff
@@ -7,19 +7,34 @@
  * function calculates average age only for men who died in this century
  *
  * To calculate century:
- * Divide year of person's death by 100: Math.ceil(person.died / 100)
+ * Divide year of personson's death by 100: Math.ceil(personson.died / 100)
  *
  * @param {object[]} people
  * @param {number} century - optional
  *
  * @return {number}
  */
+
+function averageAge(persons) {
+  return persons.map(person => person.died - person.born)
+    .reduce((firstperson, secondperson) =>
+      firstperson + secondperson) / persons.length;
+};
+
 function calculateMenAverageAge(people, century) {
   // write code here
   // learn how to use array methods like .filter .map .some .every .find .reduce
   // avoid using loop and forEach
   // replace `if ()` statement with &&, || or ?:
   // without nesting
+  const everyMan = people.filter(person => person.sex === 'm');
+  const thisCenturyPeople = people.filter(person =>
+    Math.ceil(person.died / 100) === century
+    && person.sex === 'm');
```

Comment:
Checklist point 2: if you move your shortened return statement in arrow function to a newline, use round brackets. Also, you could filter `everyMan` array here and avoid extra check on the sex
```suggestion
  const thisCenturyPeople = everyMan.filter(person => (
    Math.ceil(person.died / 100) === century
  ));
```

Reply:
will fix it